File: Backend/Drugs-Microservice/src/main/java/com/drugs/models/Drugs.java
package com.drugs.models;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Document(collection="drugs")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Drugs {
	@Id
	private String drugsId;
	private String drugsName;
	private double drugsCost;
	private int stockQty;
	private String categories;
	private String drugsDescription;
	private String supplierName;
	
	

}

File: src/app/models/drugs.ts
import { FileHandle } from "./file-handle";

export class Drugs {
    constructor(
        public drugsId:string,
        public drugsName: string,
        public drugsDescription: string,
        public drugsCost: number,
        public stockQty: number,
        public categories: string,
        public supplierName: string,
        public dateOfExpiration: string,
        public productImages: FileHandle[]
    ){}


}
